### PR TITLE
[PW-810] Add version controller

### PR DESCRIPTION
**Pull request by florfrioli**
## Summary

- Added api version controller
- Added version spec
- Fixed rubocop Rspec validation

- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] I have checked if my change needs adjustments in Active Admin
- [X] I have checked if my change needs adjustments in the API documentation
- [X] I have reviewed the files changed before releasing the pull request
- [X] I have updated the JIRA card associated with: column status, incurred hours and a comment for the QA team.
- [ ] BEFORE MERGE: I have updated the JIRA card column before merging the pull request

## Screenshots

image
image

## JIRA Card

[Link to the associated JIRA card.](https://widergy.atlassian.net/browse/PW-810)

### CODE REVIEW

File: spec/version_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  subject(:version) { described_class::VERSION }
+
+  it 'has a version number' do
+    expect(version).not_to be nil
+  end
+
+  it 'has the correct version number' do
+    expect(version).to eq('1.0.0')
+  end
+end
```
Comment (marciondg) [inline spec/version_spec.rb:13]: `versions_controller_spec.rb` is missing, right?

Comment (florfrioli) [reply spec/version_spec.rb:13]: Done!

**Review by marciondg — APPROVED** [review]
Let's go chaca!

**Review by jalvarezramirez — APPROVED** [review]
Go chaca!

**Pull request merged**